### PR TITLE
Add a couple more links to Parliamentary stuff

### PR DESCRIPTION
**Pull request by zuzak**

### CODE REVIEW

File: modules/division.js [hunk 1]
```diff
@@ -32,9 +32,9 @@ module.exports = {
         return `moment of interruption is ${dateOfInterruption.fromNow()} (${dateOfInterruption.format('h:mm a')})`
       }
     },
-    // these two are only valid for the 2017 to 2019 Commons session:
+    // these following are only valid for the 2017 to 2019 Parliamentary session:
```
Comment (danhedron) [inline modules/division.js]:
```suggestion
    // the following are only valid for the 2017 to 2019 Parliamentary session:
```

File: modules/division.js [hunk 2]
```diff
@@ -47,6 +47,22 @@ module.exports = {
...
+    },
+    future: {
+      help: 'generates link to Commons future business',
+      aliases: [ 'futurebusiness' ],
+      command: () => {
+        const ts = moment().format('YYMMDD')
+        return `https://publications.parliament.uk/pa/cm201719/cmagenda/fb${ts}.htm`
+      }
+    },
+    lbusiness: {
```
Comment (danhedron) [inline modules/division.js]: `business [commons|lords]` default to commons?

**Comment by zuzak** [comment]
how can the coverage increase I'M ADDING MORE STUFF

File: modules/division.js (hunk 2 above)
Comment (zuzak) [reply modules/division.js]: Probably a good idea but we don't support optional arguments nicely at the moment, so we'd have to do an `if (msg.args && msg.args[0] && msg.args[0].toLowerCase() === 'lords')

where's the U+XXXX PERSON ADDING TICKET TO BACKLOG emoji

**Pull request merged**